### PR TITLE
@segiddins => Add data as a place for finding errors from trunk

### PR DESCRIPTION
**Pull request by orta**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.5.1
```
Comment (segiddins) [inline CHANGELOG.md]: `Master`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+##### Enhancements
```
Comment (segiddins) [inline CHANGELOG.md]: `Bug Fixes`

**Comment by segiddins** [comment]
I'll merge as soon as I've got tests ready to add, shouldn't be long

**Comment by segiddins** [comment]
@kylef want to take a quick look at this?

**Comment by orta** [comment]
nice, looks way better than my first shot

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+## Master
+
+##### Bug Fixes
+
+* Fix the detection of spec validation errors, and present the proper error
+  (and messages) to the user.  
+  [Orta Therox](https://github.com/orta)
+  [#40](https://github.com/CocoaPods/cocoapods-trunk/issues/40)
```
Comment (kylef) [inline CHANGELOG.md]: This should be #39, #40 is the pull request.

**Comment by orta** [comment]
:+1:

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+## Master
+
+##### Bug Fixes
+
+* Fix the detection of spec validation errors, and present the proper error
+  (and messages) to the user.  
+  [Orta Therox](https://github.com/orta)
+  [#40](https://github.com/CocoaPods/cocoapods-trunk/issues/39)
```
Comment (segiddins) [inline CHANGELOG.md]: `#39`?

**Pull request merged**